addons/account/tests/test_chart_template.py

```diff
@@ -318,6 +318,45 @@ def local_get_data(self, template_code):
             {'name': 'Tax 4'},
         ])
 
+    def test_remove_fiscal_position_try_loading_force_create_false(self):
+        """Test that removing a fiscal position and calling try_loading with force_create=False does not recreate it."""
+
+        # Ensure the fiscal position exists
+        fiscal_position = self.env['account.fiscal.position'].search([
+            ('company_id', '=', self.company.id),
+            ('name', '=', 'Fiscal Position')
```

Comment:
```suggestion
            ('name', '=', 'Fiscal Position'),
```

Same at other places

Reply:
:eyes: